pkg/controller/installation/installation_controller.go

```diff
@@ -671,33 +674,33 @@ func (r *ReconcileInstallation) preflightChecks(installation *integreatlyv1alpha
 
 				return reconcile.Result{}, err
 			}
-			logrus.Infof("found required secret: %s", secretName)
+			log.Infof("found required secret", l.Fields{"secret": secretName})
 			eventRecorder.Eventf(installation, "Normal", integreatlyv1alpha1.EventPreflightCheckPassed,
 				"found required secret: %s", secretName)
 		}
 	}
 
-	logrus.Infof("getting namespaces")
+	log.Info("getting namespaces")
 	namespaces := &corev1.NamespaceList{}
 	err := r.client.List(context.TODO(), namespaces)
 	if err != nil {
 		// could not list namespaces, keep trying
-		logrus.Infof("error listing namespaces, will retry")
+		log.Info("error listing namespaces, will retry")
```

Comment:
Not sure if the `will retry` bit is needed, just seems like everything in our reconcilers is going to be `retried`, maybe an error or stay consistent with other logs (`going to retry` or not for all logs that might have trigger on failure)?

Reply:
Updated to: `log.Warningf("error listing namespaces", l.Fields{"error": err.Error()})`